### PR TITLE
Fix -0 attempt 2

### PR DESCRIPTION
**Pull request by JacobBlomgren**
So it turned out that react intl does something weird to the '-' sign

### CODE REVIEW

**Comment by Gysis** [comment]
the -0 check seems a bit weird. Isn't there a sign() function you can use?

**Comment by JacobBlomgren** [comment]
> isMinusZero = val === 0 && Math.sign(val) === -1
> 
> i thoink would be more clear

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sign
> Math.sign(-0);    // -0
I'm afraid not

File: src/Molecules/Number/Number.tsx [hunk 1]
```diff
@@ -108,10 +108,11 @@ const NumberComponent: NumberComponentType = ({
     getNumberOptions(value, { ticks, decimals, minimumDecimals, maximumDecimals }),
   );
 
+  const isMinusZero = (val: number) => 1 / val === -Infinity;
```
Comment (shmidt-i) [inline src/Molecules/Number/Number.tsx]: Is it legal is Sweden ? 😂

Comment (JacobBlomgren) [reply src/Molecules/Number/Number.tsx]: what do you mean?

Comment (shmidt-i) [reply src/Molecules/Number/Number.tsx]: dividing by zero :) anyway, it feels like a decent check here :)

**Pull request merged**